### PR TITLE
Refs #NAVIO-648 ~ Service Navitia component implemented.

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR add [Navitia Component](https://github.com/CanalTP/NavitiaComponent) and [Navitia Profiler](https://github.com/CanalTP/NavitiaProfilerBundle) in configuration of composer.

Service "canal_tp.navitia" is declared.

# How to test it ?

### 1. Add "canaltp/navitia-bundle" in your composer.json file

    composer require "canaltp/navitia-bundle"

### 2. Add configuration in your config.yml

    canal_tp_navitia:
        configuration:
            url: 'http://api.navitia.io'
            token: YOUR_TOKEN
            format: object

### 3. Do request

    $query = array(
        'api' => 'coverage',
        'parameters' => [
            'region' => 'centre'
        ]
    );
    coverages = $this->get('canal_tp.navitia')->call($query);

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "email": "ludovic.roche@canaltp.fr"
+        },
+        {
+            "name": "Rémy Abi Khalil",
+            "email": "remy.abikhalil@canaltp.fr"
+        }
+    ],
+    "require": {
+        "canaltp/navitia": "1.2.*",
+        "canaltp/navitia-profiler-bundle": "^0.0.2"
```
Comment (alcalyn) [inline composer.json]: unused dependency

**Comment by alcalyn** [comment]
Missing bundle registration in AppKernel because not working automatically

**Comment by alcalyn** [comment]
missing license

**Comment by alcalyn** [comment]
Missing dependencies:

        "symfony/config": "~2.3",
        "symfony/dependency-injection": "~2.3",
        "symfony/http-kernel": "~2.3",

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        ->children()
+            ->arrayNode('configuration')
+                ->info('Navitia configuration')
+                ->children()
+                    ->scalarNode('url')
+                        ->cannotBeEmpty()
+                        ->isRequired()
+                    ->end()
+                    ->scalarNode('format')
+                        ->defaultValue('object')
```
Comment (lrocheWB) [inline DependencyInjection/Configuration.php]: Three possible cases: json, object (cf. [Readme](https://github.com/CanalTP/NavitiaComponent/blob/master/README.md))
So, I will add:
```
->info('This value accepts: json or object.')
```
According to file below, "xml" is an valid option. Why it's not in Readme file:
https://github.com/CanalTP/NavitiaComponent/blob/68e1fcb906b738c282154abafe8bb1843834efb7/Tests/Service/ServiceFacadeTest.php#L26

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+        'parameters' => [
+            'region' => 'centre'
+        ]
+    );
+    $coverages = $this->get('canal_tp.navitia')->call($query);
+
+Contributing
+-------------
+
+1. Rémy Abi-khalil <https://github.com/netmisa>
```
Comment (lrocheWB) [inline README.md]: My opinion, a simple link is better (non-blocking for merger):
```
Contributing
------------
[View all contributors](https://github.com/CanalTP/NavitiaBundle/graphs/contributors)
```

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (Netmisa) [inline DependencyInjection/Configuration.php]: Thanks !

**Pull request merged**